inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp

```diff
@@ -111,7 +111,8 @@ bool ngraph::pass::ConvertOpSet1ToLegacy::run_on_function(std::shared_ptr<ngraph
     manager.register_pass<ngraph::pass::ConstantFolding>();
 
     // Multiply the thrird and fourth input instead of the output of FQ with all const inputs
-    manager.register_pass<ngraph::pass::FakeQuantizeMulFusion>();
+    // EDIT: skip pass in order to fix accuracy
+    // manager.register_pass<ngraph::pass::FakeQuantizeMulFusion>();
```

Comment:
Does it means that we have a bug in the transformation pass? Or should we disable it for some cases?

Reply:
it was deleted in master branch, but when release branch for kmb plugin was fixed, the pass was still there.

Reply:
I realized that we removed it on master so this PR is just a cherry-pick to kmb branch.